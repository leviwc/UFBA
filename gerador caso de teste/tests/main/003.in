29 372
15 24
3 23
11 2
2 17
11 27
18 1
6 24
12 12
24 5
7 16
19 10
17 7
9 21
20 1
5 20
13 11
27 26
13 2
13 18
19 18
19 19
12 13
21 15
15 19
8 24
12 27
4 1
7 10
9 2
22 15
28 17
15 23
14 19
4 11
18 12
5 21
16 1
11 12
16 14
23 24
20 16
22 17
26 24
22 6
23 26
21 10
28 24
8 23
27 10
9 8
7 12
4 23
19 8
21 16
29 7
11 19
25 7
8 14
17 13
27 7
10 11
5 27
7 23
18 14
29 25
19 24
9 24
22 28
28 29
25 4
18 11
25 26
5 26
5 22
1 28
6 13
6 2
25 22
29 11
24 14
11 26
4 26
18 5
16 4
11 5
2 28
23 28
10 24
19 3
5 7
20 11
4 3
21 25
26 13
8 5
5 19
25 20
23 21
8 11
29 23
20 29
14 10
6 21
12 1
2 21
2 19
27 18
14 4
23 16
28 3
6 5
2 5
19 16
23 27
10 4
3 21
23 2
22 18
7 11
6 25
8 25
15 3
17 17
20 2
13 24
26 17
11 11
4 18
5 14
15 12
26 10
16 6
10 1
2 1
12 21
16 16
22 11
16 8
8 28
18 16
19 21
22 2
3 14
16 27
7 4
12 19
29 9
8 29
22 19
15 17
29 3
20 6
18 2
8 17
4 28
23 11
7 28
17 18
9 12
13 7
29 12
1 26
6 1
20 17
20 14
7 8
27 28
23 12
29 1
16 5
17 1
21 20
5 13
3 18
22 26
2 4
25 18
25 9
12 26
13 21
28 6
13 16
25 5
3 8
26 16
5 1
15 20
18 26
4 4
1 15
18 21
13 8
27 2
9 14
14 13
2 29
28 11
10 20
29 6
29 15
12 3
26 19
27 20
18 9
8 22
17 6
15 26
5 5
24 3
21 22
10 8
4 5
2 12
12 6
8 8
25 24
14 11
17 10
19 7
28 13
24 1
22 7
15 14
22 9
24 16
17 14
17 23
12 20
10 28
17 11
1 22
24 11
9 16
29 18
23 22
24 18
29 26
28 12
18 20
1 14
10 23
1 11
27 8
6 19
20 23
17 27
21 4
18 18
4 19
3 11
4 29
27 13
8 21
10 25
27 27
7 3
20 26
13 10
4 27
5 28
12 5
3 20
24 4
15 27
4 9
14 27
24 21
5 10
6 10
25 27
9 28
18 15
11 21
17 4
23 18
9 17
3 16
28 21
25 17
2 3
3 3
6 18
27 9
14 7
18 28
2 14
23 5
2 26
25 15
25 28
8 15
20 8
16 28
4 20
23 9
21 14
6 3
18 8
9 19
10 10
4 8
15 13
11 15
2 24
11 16
27 21
3 27
18 10
17 29
13 9
19 13
29 24
10 16
26 8
1 1
24 12
7 15
8 6
3 22
25 25
22 20
16 2
4 6
5 17
9 1
25 23
7 7
28 20
17 21
20 9
9 11
3 13
7 6
12 16
27 19
8 1
22 4
1 23
29 27
2 15
14 23
24 20
1 27
12 10
22 29
28 14
25 12
22 24
3 10
19 1
20 13
1 7
9 7
29 10
5 29
20 20
21 7
9 6
7 2
6 11
14 25
9 26
14 14
19 28
17 19
25 3
17 3
29 14
12 17
5 3
5 15
14 6
