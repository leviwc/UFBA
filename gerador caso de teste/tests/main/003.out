Resultado com busca binaria: 3
3209.431000 s
Resultado com backtracking apenas: 3
0.031000 s
Resultado testando todas as possibilidades: 3
18794.011000 s
Resultado com busca binaria otimizada: 3
63.122000 s
